src/Sylius/Bundle/AdminBundle/Resources/config/grids/country.yml

```diff
@@ -0,0 +1,31 @@
+sylius_grid:
+    grids:
+        sylius_admin_country:
+            driver:
+                name: doctrine/orm
+                options:
+                    class:  %sylius.model.country.class%
+            sorting:
+                code: asc
+            fields:
+                code:
+                    type: string
+                    label: sylius.ui.code
+                name:
+                    type: twig
+                    label: sylius.ui.name
+                    path: .
+                    options:
+                        template: SyliusAdminBundle:Country/Fields:_name.html.twig
+                enabled:
+                    type: twig
+                    label: sylius.ui.enabled
+                    options:
+                        template: SyliusAdminBundle:Grid/Fields:_enabled.html.twig
```

Comment:
I renamed this to singular ``Field`` and removed the ``_`` prefix. Please adjust. :)